creed/qc_comp_test.py

```diff
@@ -0,0 +1,331 @@
+import numpy as np
+import pandas as pd
+from requests.exceptions import HTTPError
+import xarray as xr
+
+from creed.mibitracker_utils import MibiRequests
+from creed import qc_comp
+import ark.utils.io_utils as io_utils
+import ark.utils.misc_utils as misc_utils
+import ark.utils.test_utils as test_utils
+
+import os
+import pytest
+import tempfile
+
+
+RUN_POINT_NAMES = ['Point%d' % i for i in range(1, 13)]
+RUN_POINT_IDS = list(range(661, 673))
+
+# NOTE: all fovs and all channels will be tested in the example_qc_metric_eval notebook test
+FOVS_CHANS_TEST_MIBI = [
+    (None, ['CCL8', 'CD11b'], None, RUN_POINT_NAMES, RUN_POINT_IDS),
+    (None, ['CCL8', 'CD11b'], "TIFs", RUN_POINT_NAMES, RUN_POINT_IDS),
+    (['Point1'], None, None, RUN_POINT_NAMES[0:1], RUN_POINT_IDS[0:1]),
+    (['Point1'], None, "TIFs", RUN_POINT_NAMES[0:1], RUN_POINT_IDS[0:1]),
+    (['Point1'], ['CCL8', 'CD11b'], None, RUN_POINT_NAMES[0:1], RUN_POINT_IDS[0:1]),
+    (['Point1'], ['CCL8', 'CD11b'], "TIFs", RUN_POINT_NAMES[0:1], RUN_POINT_IDS[0:1])
+]
+
+
+FOVS_CHANS_TEST_QC = [
+    (None, None, False),
+    (None, None, True),
+    (['fov0', 'fov1'], None, False),
+    (['fov0', 'fov1'], None, True),
+    (None, ['chan0', 'chan1'], False),
+    (None, ['chan0', 'chan1'], True),
+    (['fov0', 'fov1'], ['chan0', 'chan1'], False),
+    (['fov0', 'fov1'], ['chan0', 'chan1'], True)
+]
+
+MIBITRACKER_EMAIL = 'qc.mibi@gmail.com'
+MIBITRACKER_PASSWORD = 'The_MIBI_Is_Down_Again1!?'
```

Comment:
sigh

Reply:
i think the long term solution for this type of problem is to just make this an environment variable.  that can be set in travis pretty easily I think, and as for local testing, we just require you set that env variable before testing.

Reply:
Yup. We can also use notebook injection so the password isn't visible here, it could just be a placeholder.